### ISSUE TITLE
Refactor tools

### ISSUE THREAD

**Issue by antoninoLorenzo** [entropy-research/Devon#27]
By looking inside `devon_agent` folder it comes out that there is `tools.py` module and `tools` folder module, I don't know what are specifically the redundant parts in the first one, however I think it should be checked and `tools.py` should be removed, mainly for code maintainability.

**Comment by Mihir1003** [comment]
yes, we are migrating to use tool.py and tools. tools.py was the old tool interface. Will remove it.
